### PR TITLE
Add WF for 2024 PbPb skim

### PR DESCRIPTION
**Pull request by youyingli**
#### PR description:
Add the workflow for the 2024 PbPb skim with MC generation inside relval to try to solve the issue #46586

@mmusich @AdrianoDee @mandrenguyen 

#### PR validation:

```runTheMatrix.py -w "standard" -l 143.201,143.202```

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Will make a backport to 14_1_X if passd.

### CODE REVIEW

**Comment by AdrianoDee** [comment]
please test workflows 143.201,143.202

File: Configuration/PyReleaseValidation/python/relval_standard.py [hunk 1]
```diff
@@ -559,6 +559,10 @@
 workflows[142.901] = ['',['RunUPC2023','RECODR3_2024_UPC','HARVESTDPROMPTR3']]
 workflows[142.902] = ['',['RunUPC2023','RECODR3_2024_HIN','HARVESTDPROMPTR3']]
 
+### run3-2024 skim (2024 HI MC temp)
+workflows[143.201] = ['',['HydjetQ_B12_5362GeV_2024','DIGIHI2024','SKIMHIFORWARDRUN3_2024','HARVESTHI2024']]
+workflows[143.202] = ['',['HydjetQ_B12_5362GeV_2024','DIGIHI2024','SKIMHIPHYSICSRAWPRIMERUN3_2024','HARVESTHI2024']]
```
Comment (mmusich) [inline Configuration/PyReleaseValidation/python/relval_standard.py:564]: hmm, I think the rawprime wf should use the rawprime reco. 
see 
https://github.com/cms-sw/cmssw/blob/15283d2354f36562a6adc97318849ece08035b5b/Configuration/PyReleaseValidation/python/relval_standard.py#L874

Comment (youyingli) [reply Configuration/PyReleaseValidation/python/relval_standard.py]: OK, let's follow this wf.

**Comment by srimanob** [comment]
test parameters:
- workflows = 143.201,143.202
- relvals_opt = --what cleanedupgrade,standard,highstats,pileup,generator,extendedgen,production,identity,ged,machine,premix,nano,gpu,2017,2026

**Comment by srimanob** [comment]
@cmsbuild please test

File: Configuration/PyReleaseValidation/python/relval_standard.py (hunk 1 above)
Comment (mmusich) [reply Configuration/PyReleaseValidation/python/relval_standard.py]: Well,  if you run `SKIMHIPHYSICSRAWPRIMERUN3_2024` as defined right now, you are not using the strip approximate cluster modifier used in `RECOHI2024PROD`:

https://github.com/cms-sw/cmssw/blob/ce4d0b1395f39ad6ea81b98cd4dab57c3ac4df60/Configuration/PyReleaseValidation/python/relval_steps.py#L3261-L3266

then's pointless to run the `RAWPRIMESIMHI18` step in between as you'll just consume the regular raw and not rawprime.

Comment (youyingli) [reply Configuration/PyReleaseValidation/python/relval_standard.py]: My bad. Now it's added. If it's correct, please test.

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by AdrianoDee** [comment]
+pdmv

The new wfs test the 2024 HI skims and produce the expected outputs.

**Comment by Moanwar** [comment]
+Upgrade

**Comment by mandrenguyen** [comment]
+1

**Pull request merged**